src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { LoadingError } from '../../components/LoadingError';
+import { PeopleTable } from '../../components/PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [loadingError, setLoadingError] = useState(false);
```

Comment:
Rename boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isLoading, setIsLoading] = useState(false);
  const [isLoadingError, setIsLoadingError] = useState(false);
```